tools/topology/sof-apl-dmic-b2ch.m4

```diff
@@ -66,11 +66,8 @@ dnl DAI_CONFIG(type, dai_index, link_id, name, ssp_config/dmic_config)
 
 DAI_CONFIG(DMIC, 1, 7, NoCodec-7,
 	   dnl DMIC_CONFIG(driver_version, clk_min, clk_mac, duty_min, duty_max,
-	   dnl		   sample_rate, fifo word length, type, dai_index,
-	   dnl             pdm controller config)
+	   dnl		   sample_rate, fifo word length, unmute time, type,
+	   dnl              dai_index, pdm controller config)
```

Comment:
misaligned

Reply:
yep (responding to mark to myself I've done this and not forgetting something...)